### PR TITLE
update readme: add missing packed_struct lib

### LINKED ISSUE #6: Add packed_struct to list of similar crates
As far as I understand this crate does not only deal with endianess but also with packing and unpacking structs. To me it looks like Rust has quite a weak spot in this domain, because there is really not one widely used/well maintained crate for (de)serializing structs to a custom byte format. Having one more mentioned in your list will hopefully spare some time for someone in the future. 

https://crates.io/crates/packed_struct

### PR DESCRIPTION
**Pull request by wucke13**
Fixes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +83,7 @@ variables in the desired endian format.
 [bincode]:https://crates.io/crates/bincode
 [byteorder]:https://crates.io/crates/byteorder
 [bytes]:https://crates.io/crates/bytes
+[packed_struct]:https://docs.rs/packed_struct/0.3.0/packed_struct/
```
Comment (xoac) [inline README.md]: All links are to crates.io. Could you change it too?

Comment (wucke13) [reply README.md]: Sure!

**Comment by xoac** [comment]
Thank you!

**Comment by xoac** [comment]
Oh I merged this and forget I use [cargo-readme](https://github.com/livioribeiro/cargo-readme). Maybe you find it useful - changes should be done to `src/lib.rs` and README.md should be generated with cargo-readme.

**Comment by wucke13** [comment]
Ah, that's a neat tool! Thank you too!

**Pull request merged**